FRENCH/src/ch07-03-paths-for-referring-to-an-item-in-the-module-tree.md

```diff
@@ -0,0 +1,883 @@
+<!--
+## Paths for Referring to an Item in the Module Tree
+-->
+
+## Les chemins pour désigner un élément dans l'arborescence de module
+
+<!--
+To show Rust where to find an item in a module tree, we use a path in the same
+way we use a path when navigating a filesystem. If we want to call a function,
+we need to know its path.
+-->
+
+Pour indiquer à Rust où trouver un élément dans l'arborescence du module, nous
+utilisons un chemin à l'instar des chemins que nous utilisons lorsque nous
+naviguons dans un système de fichiers. Si nous voulons appeler une fonction,
+nous avons besoin de savoir son chemin.
+
+<!--
+A path can take two forms:
+-->
+
+Un chemin peut avoir deux formes :
+
+<!--
+* An *absolute path* starts from a crate root by using a crate name or a
+  literal `crate`.
+* A *relative path* starts from the current module and uses `self`, `super`, or
+  an identifier in the current module.
+-->
+
+* Un *chemin absolu* qui commence à partir de la racine de la crate en utilisant
+  le nom d'une crate, ou le mot `crate`.
+* Un *chemin relatif* qui commence à partir de notre module courant et qui
+  utilise `self`, `super`, ou un identificateur à l'intérieur du module.
+
+<!--
+Both absolute and relative paths are followed by one or more identifiers
+separated by double colons (`::`).
+-->
+
+Les deux chemins absolus et relatifs sont suivis par un ou plusieurs
+identificateurs séparés par des doubles deux-points (`::`).
+
+<!--
+Let’s return to the example in Listing 7-1. How do we call the
+`add_to_waitlist` function? This is the same as asking, what’s the path of the
+`add_to_waitlist` function? In Listing 7-3, we simplified our code a bit by
+removing some of the modules and functions. We’ll show two ways to call the
+`add_to_waitlist` function from a new function `eat_at_restaurant` defined in
+the crate root. The `eat_at_restaurant` function is part of our library crate’s
+public API, so we mark it with the `pub` keyword. In the [”Exposing Paths with
+the `pub` Keyword”][pub]<!-- ignore -- > section, we’ll go into more detail
+about `pub`. Note that this example won’t compile just yet; we’ll explain why
+in a bit.
+-->
+
+Reprenons notre exemple de l'encart 7-1. Comment utiliserions-nous la fonction
+`ajouter_a_la_liste_attente` ? Cela revient à demander : quel est le chemin de
+la fonction `ajouter_a_la_liste_attente` ? Dans l'encart 7-3, nous avons un peu
+simplifié notre code en enlevant quelques modules et quelques fonctions. Nous
+allons voir deux façons d'appeler la fonction `ajouter_a_la_liste_attente` à
+partir d'une nouvelle fonction `manger_au_restaurant` définie dans la crate
```

Comment:
Hum ... You mean ```une nouvelle fonction `manger_au_restaurant` définie à racine de la crate``` instead of ```une nouvelle fonction `manger_au_restaurant` définie dans la crate``` ?

Reply:
Yes. We are talking about where the item is located in the crate. `crate racine` would imply that we are talking about another crate (?)

Reply:
Oh ok, I think it's the same :smile: